.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -333,7 +335,6 @@ onboarding_t0:
   - snmp/test_snmp_link_local.py
   - snmp/test_snmp_psu.py
   - snmp/test_snmp_queue_counters.py
-  - sub_port_interfaces/test_show_subinterface.py
```

Comment:
why it is removed here?

Reply:
We can remove it from **onboarding** T0 PR checker because it is added in T0 PR checker.